site/docs/configurable-attributes.md

```diff
@@ -526,6 +533,43 @@ sh_binary(
 
 This automatically expands the `select` to the original syntax above.
 
+### <a name="config-setting-aliasing"></a>`config_setting`Aliasing
+
+If you'd like to `OR` conditions under a proper `config_setting` that any rule
+can reference, you can use a `select`able [alias](be/general.html#alias) that
+matches any of the desired conditions:
+
+```python
+alias(
+    name = "config1_or_2_or_3",
+    actual = select({
```

Comment:
Should there be a caveat about how this will fail if there's a possibility where multiple conditions may match, but they don't refine each other? e.g., --cpu=x86 and --compiler=gcc?

Reply:
Done!